### ISSUE TITLE
Compile error on Fedora 34

### ISSUE THREAD

**Issue by polemon** [skyjake/lagrange#365]
I'm getting this error when trying to compile on Fedora 34:

```
/home/bereziak/make/lagrange/src/gmdocument.c:1911:52: error: implicit declaration of function ‘isnumber’ [-Werror=implicit-function-declaration]
 1911 |                 else if (size_Range(&line) >= 2 && isnumber(line.start[0]) &&
      |                                                    ^~~~~~~~
```

All libraries (and their `-devel` counterparts) are installed as far as I can tell.

**Comment by skyjake** [comment]
This is with the currently unreleased dev branch? Mea culpa, that should be using the standard `isdigit`...

**Comment by skyjake** [comment]
Fixed in 066bb85e2169daacb52a4b7c71923de9468a1ef3.

**Comment by polemon** [comment]
Yes, I `git clone`'d a minute ago
